RecoPixelVertexing/PixelTriplets/plugins/CAHitNtupletGeneratorKernelsImpl.h

```diff
@@ -700,7 +716,7 @@ __global__ void kernel_sharedHitCleaner(TrackingRecHit2DSOAView const *__restric
                                         HitContainer const *__restrict__ ptuples,
                                         TkSoA const *__restrict__ ptracks,
                                         Quality *__restrict__ quality,
-                                        uint16_t nmin,
+                                        int nmin,
```

Comment:
out of curiosity, why 16 -> 32 bits ?
do we need the wider range, or it just doesn't make a difference ?

Reply:
there was no reason to be a ```uint16_t``` in first place. sticking with ```int``` in particular if operations are performed is safer (and makes eesier to deal with all comparisons).
```uint``` shall be used only if bit operations are required.